### PR TITLE
Hls 2.0.0.1 forward port

### PR DESCRIPTION
**Pull request by hasufell**

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Releasing
+
+## Release checklist
```
Comment (fendor) [inline RELEASING.md:3]: I love this addition!

This has a big overlap with https://github.com/haskell/haskell-language-server/blob/master/docs/contributing/releases.md, we should probably merge these documents.

Comment (hasufell) [reply RELEASING.md:3]: Possibly... but the other document is old and I've never looked at it.

My suggestion is to have another contributor do a release while I'm available and ask questions when anything in the checklist is unclear... and then we can add more information as we go.

Comment (michaelpj) [reply RELEASING.md:3]: An acceptable merge operation is "discard the old one" :)

Comment (fendor) [reply RELEASING.md:3]: The old list is at least partially up-to-date and contains a lot of detailed information, I would not just delete it.

**Comment by hasufell** [comment]
who keeps pushing to master?

**Comment by michaelpj** [comment]
It's called PR's getting merged ;)

**Pull request merged**